### PR TITLE
WebUI Design update

### PR DESCRIPTION
**Pull request by VicDeo**
Ref #250

### CODE REVIEW

**Comment by VicDeo** [comment]
sample

**Comment by karlitschek** [comment]
:+1:

**Comment by jancborchardt** [comment]
@VicDeo how can I invoke this mode to test it? Alternatively if it’s faster you could do the additional changes, just a few details:

- [ ] remove the grey `background-color: #f7f7f7;` from the h3
- [ ] add a `margin-top:10px` to the h3 for whitespace to separate
- [ ] indent the text »Current version …« and »No updates found …« a bit more to the right to be correctly aligned with the other text above and below
- [ ] move the recheck box so it’s not floating so far on the right, it can be directly next to the text
- [ ] ideally use h2 instead of h3 so the styles from core for the heading is used. If the core styles are not loaded there, just use this:
```
font-size: 20px;
margin-bottom: 12px;
font-weight: 300;
```

Thanks! :)

**Comment by VicDeo** [comment]
@jancborchardt 
You can either install a daily build or clone this repo into your OC master repo 
In the latter case you need manually set channel to daily with
`sudo -u wwwrun php occ config:app:set core OC_Channel --value daily`

(because the daily channel always reports there are new updates)

**Comment by VicDeo** [comment]
@jancborchardt Like this?

updater-init
updater-progress

File: pub/css/main.css [hunk 1]
```diff
@@ -727,10 +726,8 @@ li.step{
 	opacity: .57;
 }
 
-li.step h3 {
-	background-color: #f7f7f7;
+li.step h2 {
 	padding: 5px 2px 5px 30px;
-	border: 1px solid #ddd;
 	-ms-filter: "progid:DXImageTransform.Microsoft.Alpha(Opacity=57)";
 	opacity: .57;
```
Comment (jancborchardt) [inline pub/css/main.css]: Here, just add:
```
margin-top: 12px;
margin-bottom: 0;
```
That should fix the whitespace.

File: pub/css/main.css [hunk 2]
```diff
@@ -740,26 +737,26 @@ li.current-step, li.passed-step, li.failed-step{
 	opacity: 1;
 }
 
-li.step h3{
+li.step h2{
 	background-position:8px 50%;
 	background-repeat: no-repeat;
 }
```
Comment (jancborchardt) [inline pub/css/main.css]: You can just move these rules into the li.step h2 block above. :)

**Comment by jancborchardt** [comment]
NICE! Almost perfect. See the inline comments for the whitespace fix (as you see, there is too much whitespace below the heading, but not above – it needs to be the other way around so the content of below the heading actually belongs to it visually. :)

**Comment by VicDeo** [comment]
@jancborchardt 
updater-new

**Comment by jancborchardt** [comment]
Looks damn awesome now! Great job @VicDeo! :+1: 

cc @owncloud/designers @jospoortvliet for a nice example of a design evolution. :)

**Pull request merged**